### PR TITLE
Custom Colors: update luminance and contrast formulae

### PR DESCRIPTION
**Pull request by jeherve**
in accordance with latest WCAG 2.0 standards.

Merges r128444-wpcom.

### CODE REVIEW

File: _inc/lib/class.color.php [hunk 1]
```diff
@@ -419,7 +419,7 @@ public function toXyz()
 		}
 		$rgb = $rgb_new;
 
-		//Observer. = 2�, Illuminant = D65
+		//Observer. = 2�, Illuminant = D65
```
Comment (eliorivero) [inline _inc/lib/class.color.php]: It doesn't affect anything, but this character shouldn't be like this. It's also like this in the r128444-wpcom.

File: _inc/lib/class.color.php [hunk 2]
```diff
@@ -547,7 +551,7 @@ public function toLuminosity() {
 	 * @param  Jetpack_Color  $color Another color
 	 * @return float
 	 */
-	public function getDistanceLuminosityFrom(Jetpack_Color $color) {
+	public function getDistanceLuminosityFrom( Jetpack_Color $color) {
```
Comment (eliorivero) [inline _inc/lib/class.color.php]: Needs space towards right.

**Comment by eliorivero** [comment]
Apart from that character, lgtm. Tested with @jeherve's Color Posts plugin and it works fine.

**Comment by jeherve** [comment]
Fixed!

**Comment by zinigor** [comment]
@jeherve why is 09cd983 not part of this PR? Can you please check where you committed that to?

**Pull request merged**